### PR TITLE
Update compy_next.sh to use intel instead of pgi

### PR DESCRIPTION
**Pull request by wadeburgess**

### CODE REVIEW

File: jenkins/compy_next.sh [hunk 1]
```diff
@@ -5,4 +5,5 @@ export SCRIPTROOT=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && cd .. && pwd )
 export CIME_MACHINE=compy
 source $SCRIPTROOT/util/setup_common.sh
 
-$RUNSCRIPT --compiler pgi
+#$RUNSCRIPT --compiler pgi
+$RUNSCRIPT
```
Comment (rljacob) [inline jenkins/compy_next.sh]: You have to add the 2 lines above to switch to intel?

Comment (wadeburgess) [reply jenkins/compy_next.sh]: intel is default since it is listed first in config_machines.xml, correct? so no need to specify in this script?

Comment (jgfouca) [reply jenkins/compy_next.sh]: I think @rljacob is suggesting that you delete the comment. We already have git for looking back in history, so there's no need to leave it in.

Comment (rljacob) [reply jenkins/compy_next.sh]: Oh I see, you commented out the first line.

**Pull request merged**